library/common/extensions/filters/http/platform_bridge/c_type_definitions.h

```diff
@@ -44,3 +44,7 @@ const envoy_filter_trailers_status_t kEnvoyFilterTrailersStatusStopIteration =
 // See comment above.
 const envoy_filter_trailers_status_t kEnvoyFilterTrailersStatusResumeIteration =
     kEnvoyFilterTrailersStatusContinue - 1;
+
+// These values don't exist in Envoy and are essentially arbitrary.
+const envoy_filter_resume_status_t kEnvoyFilterResumeStatusStopIteration = 1;
+const envoy_filter_resume_status_t kEnvoyFilterResumeStatusResumeIteration = -1;
```

Comment:
If these are truly arbitrary, why not start zero-indexed? I assume the `-1` is coming from the same reasoning above, in which case we should probably make that clearer here

Reply:
I simply set them to be consistent with other invocations, but the comment is meant to suggest that one shouldn't rely on them being anything in particular (other than constant). They could also be zero-indexed (and I actually don't have a preference).